src/spec/validitygenerator.py

```diff
@@ -406,22 +406,27 @@ def createValidationLineForParameterIntroChunk(self, blockname, param, params, t
 
         elif self.paramIsPointer(param):
             # Handle pointers - which are really special case arrays (i.e. they don't have a length)
+            #TODO  should do something here if someone ever uses some intricate comma-separated `optional`
             pointercount = paramtype.tail.count('*')
 
+            # Treat void* as an int
+            if paramtype.text == 'void':
+                pointercount -= 1
+
             # Could be multi-level pointers (e.g. ppData - pointer to a pointer). Handle that.
             for i in range(0, pointercount):
-                asciidoc += 'a pointer to '
+                asciidoc += 'a pointer to a '
```

Comment:
Forgot to move the first "a " outside the loop. Wil fix momentarily.
Would break only non-existent `***`+ pointers; not a big deal.